redbot/core/core_commands.py

```diff
@@ -786,9 +802,16 @@ async def mydata_owner_user_deletion_level(self, ctx, level: int):
         """
         Sets how user deletions are treated.
 
+        Example:
+            - `[p]mydata ownermanagement setuserdeletionlevel 1`
+
+        **Arguments:**
+
+        - `<level>` The strictness level for user deletion. See Level guide below.
```

Comment:
I'm open for suggestions on this. The main challenge is the negotiation between RST and how discord displays it. I chose bold because it looked better on discord's side. However, RST requires a blank link to start a list if the colon is not at the end of the line I believe. 

I will put together some other options and offer them for discussion.

Reply:
This has been settled in this poll: https://forms.gle/wiY4kY82d4qZh9358